`Alert` and `Toast` Overview Update (#1720)
hashicorp/design-system: website/docs/components/alert/partials/guidelines/guidelines.md

### When not to use

> - For subtle feedback on a user’s action, consider [Toast](/components/toast).

Author: ```suggestion
- To communicate successful feedback on a user’s action, consider [Toast](/components/toast).
```

- As a dialog to confirm an action, consider [Modal](/components/modal).
- To flag new features (e.g., "In Preview", "Beta", "New", etc), consider [Badge](/components/badge).

## Types of alerts

There are three types of alerts: page, inline, and compact.

### Page

Use page-level alerts on the top of the page, between the navigation and the breadcrumb to inform users of an event that may require the user’s attention or action.

### Inline

Use inline alerts when the message is contextual or specific to the section it’s placed in.

### Compact

> Use compact alerts when a less prominent alert is needed.

Reviewer: @MelSumner This is current content on the website, and changing it is out of the scope of the ticket. I asked Andrew to re-organize some of the content (moving the "Usage" section to the top), which is in scope.
